src/services/api/DataStorageService.ts

```diff
@@ -0,0 +1,25 @@
+import {autoinject} from "aurelia-framework";
+import { IClause } from "configurations/apiSchema";
+import { LocalStorageService } from "./LocalStorageService";
+import { DEAL_CLAUSES } from "./mockData";
+
+@autoinject
+export class DataStorageService {
+
+  constructor(private localStorageService: LocalStorageService) { }
+
+  public async getAllClauses(): Promise<IClause[]> {
+    const existingData = this.localStorageService.getClausesData();
+    if (existingData) {
```

Comment:
Can do.

Just to note my preferences here.
I prefer being more explicit with my code, instead of "less code with syntax sugar" where possible.
I also prefer to use "inline variables" over direct return.

Anyway, might make sense for us to actually have more meetings about code style, to at least _decide_ on one thing over the other, as stuff like this is not easily lintable? (or is it?)

Reply:
Ah. okok. I didn't know that. 

I also like simple code but I also like not to write if statements at all if possible 🤣 (hence, that's why I suggested to use `??` instead of that `if` statement because it looks cleaner for me personally). But I can adapt, no worries. We can leave it as it is.